### ISSUE TITLE
Improve first time UX when using SDK if only screenshot feature is needed

### ISSUE THREAD

**Issue by RabieJradi** [crowdin/mobile-sdk-android#60]
**Is your feature request related to a problem? Please describe.**
I'm only interested in using the SDK because of the screenshot upload feature, but if I follow the readme description I don't see that it is required to have the real time feature enabled in order to get the first time authorisation flow running. This is the sample code in the screenshot section:

```
    Crowdin.init(applicationContext,
        CrowdinConfig.Builder()
            .withDistributionHash(your_distribution_hash)
            .withScreenshotEnabled()
            .withSourceLanguage(source_language)
            .withAuthConfig(AuthConfig(client_id, client_secret, organization_name))
            .withNetworkType(network_type)                                           // optional
            .withUpdateInterval(interval_in_millisec)                                // optional
            .build())
```

but looking at the following code in `Crowdin.kt` it becomes apparent why the flow is not executed since both features must be enabled, which is a pitfall for spending time wondering why the screenshot uploading isn't working.

```
fun authorize(activity: Activity) {
        if (!FeatureFlags.isRealTimeUpdateEnabled || !FeatureFlags.isScreenshotEnabled) {
            return
        }
        if (isAuthorized()) {
            tryCreateRealTimeConnection()
        } else {
            createAuthDialog(activity) { AuthActivity.launchActivity(activity) }
        }
    }
```
**Describe the solution you'd like**
Possible solutions
1. Update the readme and sample code
2. Don't require that both feature flags must be enabled

**Comment by andrii-bodnar** [comment]
Hi @RabieJradi, thank you for the suggestion!

The second solution will be implemented.

**Comment by andrii-bodnar** [comment]
@RabieJradi Hi, this issue was fixed in the [1.1.6](https://github.com/crowdin/mobile-sdk-android/releases/tag/1.1.6) version.

Could you please share your experience with the Crowdin Android SDK? It may help us to make it better :slightly_smiling_face:
